x/participation/keeper/msg_participate_test.go

```diff
@@ -17,31 +17,37 @@ func Test_msgServer_Participate(t *testing.T) {
 	var (
 		sdkCtx, tk, ts        = testkeeper.NewTestSetup(t)
 		auctioneer            = sample.Address()
-		sellingCoin           = sample.Coin()
-		startTime             = sdkCtx.BlockTime().Add(time.Hour * 10) // 10 hours from now
-		registrationPeriod    = time.Hour * 5                          // 5 hours before start
+		sellingCoin1          = sample.Coin()
+		sellingCoin2          = sample.Coin()
+		registrationPeriod    = time.Hour * 5 // 5 hours before start
+		startTime1            = sdkCtx.BlockTime().Add(time.Hour * 10)
+		startTime2            = time.Unix(int64((registrationPeriod - time.Hour).Seconds()), 0)
```

Comment:
If `startTime2` is only used for the test case where startTime < time(0) + Registration Period
, I would suggest avoiding numbered variable and make the name more descriptive

`startTimeLowerRegistrationPeriod`